crates/shared/src/bad_token/token_owner_finder.rs

```diff
@@ -80,6 +80,14 @@ pub struct Arguments {
     #[clap(long, env, value_parser = duration_from_seconds, default_value = "45")]
     pub blockscout_http_timeout: Duration,
 
+    // Override the default blockscout API url for this network
```

Comment:
Turning these into doc comments will cause these texts to show up in the `--help` of the binary.
```suggestion
    /// Override the default blockscout API url for this network
```